src/main.cpp

```diff
@@ -75,23 +77,26 @@ int main(int argc, char **argv) {
     return -1;
 #endif
 
-    sl::zed::InitParams params;
-    params.mode = MODE::PERFORMANCE;
+    sl::InitParameters params;
+    params.depth_mode = sl::DEPTH_MODE_PERFORMANCE;
+    params.camera_resolution = ZED_RES;
+    params.camera_fps = FPS;
 
     // Create every ZED and init them
     for (int i = 0; i < NUM_CAMERAS; i++) {
-        zed[i] = new Camera(ZED_RES, FPS, i);
+        zed[i] = new sl::Camera();
+        params.camera_linux_id = i;
 
-        ERRCODE err = zed[i]->init(params);
+        sl::ERROR_CODE err = zed[i]->open(params);
 
-        cout << "ZED N°" << i << " -> Result : " << errcode2str(err) << endl;

```

Comment:
My OpenCV creates a myriad of windows, one per frame, when the window title includes this character; maybe it's something I did wrong while building my OpenCV, but this small change fixes it.